examples/storagetransfer-ts/index.ts

```diff
@@ -0,0 +1,112 @@
+import * as gcp from "@pulumi/gcp";
+import * as gnative from "@pulumi/google-native";
+import * as storage from "@pulumi/google-native/storage/v1";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+import * as fs from "fs";
+import * as mime from "mime";
+
+const randomString = new random.RandomString("name", {
+    upper: false,
+    number: false,
+    special: false,
+    length: 5,
+});
+
+const config = new pulumi.Config("google-native");
+const project = config.require("project");
+const bucketName = pulumi.interpolate`pulumi-transfer-test-dst-${randomString.result}`;
+const transferJobName = "transfer-test";
+
+// Create a Google Cloud resource (Storage Bucket)
+const bucket = new storage.Bucket("my-bucket", {
+    name: bucketName,
+    project: project,
+});
+
+const storageTransferServiceAccount = gcp.storage.getTransferProjectServieAccount({
```

Comment:
I've just merged invokes - can you try replacing it with the native one?

P.S. `getTransferProjectServieAccount` - do we have a typo `Servie` in this GCP resource?

Reply:
Yup - https://github.com/pulumi/pulumi-gcp/blob/master/provider/resources.go#L2333

Reply:
I tried to use the new invokes but as highlighted in #119, we are not exposing the project's service-specific service accounts getters. I tried to use the generic IAM getServiceAccount but it doesn't seem to be able to enumerate the transfer service account. I verified this from the IAM console as well (although I see some service-specific service accounts for the project for other services). For now, I am leaving this in as is.